.gitignore

```diff
@@ -1,5 +1,7 @@
 vul_data
 .DS_Store
 
+
 tmp/
 !tmp/.gitkeep
+
```

Comment:
This file shouldn't have been edited.

Reply:
I think it was my editor added stuff. It was un intentional